src/Bookings.js

```diff
@@ -12,7 +12,7 @@ const Bookings = () => {
     <div className="App-content">
       <div className="container">
         <Search search={search} />
-        {/* <SearchResults results={FakeBookings} /> */}
+        {<SearchResults />}
```

Comment:
```suggestion
        <SearchResults />
```

Reply:
Curly braces not required here when you're calling the `SearchResults` component